app/controllers/api/projects.php

```diff
@@ -155,6 +161,8 @@
             $dbForProject->createCollection($key, $attributes, $indexes);
         }
 
+        call_user_func($returnDB);
```

Comment:
Not sure I fully understand what happens here.

Reply:
Once the dbPool instance has been used, we need to return it back to the database pool. `$returnDB` is a callback function that returns the database instance to the appropriate pool